<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddDirectoryDialog</class>
 <widget class="QDialog" name="AddDirectoryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>141</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Directory</string>
  </property>
  <property name="toolTip">
   <string>Add a directory to the current project</string>
  </property>
  <property name="whatsThis">
   <string>&lt;b&gt;Add Directory Dialog&lt;/b&gt;
&lt;p&gt;This dialog is used to add a directory to the current project.&lt;/p&gt;</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>&amp;File Type:</string>
     </property>
     <property name="buddy">
      <cstring>filterComboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="filterComboBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="sourceDirLabel">
     <property name="text">
      <string>&amp;Source Directory:</string>
     </property>
     <property name="buddy">
      <cstring>sourceDirPicker</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="E5PathPicker" name="sourceDirPicker" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="toolTip">
      <string>Enter the name of the directory to add</string>
     </property>
     <property name="whatsThis">
      <string>&lt;b&gt;Source Directory&lt;/b&gt;
&lt;p&gt;Enter the name of the directory to add to the current project.
 You may select it with a dialog by pressing the button to the right.&lt;/p&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="targetDirLabel">
     <property name="text">
      <string>&amp;Target Directory:</string>
     </property>
     <property name="buddy">
      <cstring>targetDirPicker</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="E5PathPicker" name="targetDirPicker" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="toolTip">
      <string>Enter the target directory for the file</string>
     </property>
     <property name="whatsThis">
      <string>&lt;b&gt;Target Directory&lt;/b&gt;
&lt;p&gt;Enter the target directory. You may select it
 with a dialog by pressing the button to the right.&lt;/p&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="recursiveCheckBox">
     <property name="toolTip">
      <string>Select, whether a recursive add should be performed</string>
     </property>
     <property name="text">
      <string>&amp;Recurse into subdirectories</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>E5PathPicker</class>
   <extends>QWidget</extends>
   <header>E5Gui/E5PathPicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>filterComboBox</tabstop>
  <tabstop>sourceDirPicker</tabstop>
  <tabstop>targetDirPicker</tabstop>
  <tabstop>recursiveCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddDirectoryDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddDirectoryDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
